.github/workflows/pr-lint.yml

```diff
@@ -14,7 +14,5 @@ jobs:
           npm install --prefix tools/linters
       - name: run linter
         run: |
-          cd ./tools/linters
-          npx --no-install markdownlint -c markdownlint.json ../../*.md ../../text/**/*.md
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          find . -type f -name '*.md' -not -path '*/node_modules/*' -not -path '*/.github/*' |\
+            xargs -I'{}' ./tools/linters/node_modules/.bin/markdownlint -c tools/linters/markdownlint.json '{}'
```

Comment:
I've explicitly chosen not to use `npx` here to make the output cleaner. 

If I have to use `npx` like before, I'll have to change into the directory (`cd`) which means every line of the linter output will be prefixed with `../../`.